### PR TITLE
[Concept Entry] Cybersecurity, Risk Management and Spear Phishing

### PR DESCRIPTION
**Pull request by realber**
### Description

This is a cybersecurity concept, risk management.

### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [ ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by SSwiniarski** [comment]
@realber, just to note that you are pushing this from your main branch, which isn't recommended. You should create forks for each separate entry.

File: content/cybersecurity/concepts/risk-management/risk-management.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+## Risk Management Strategies:
+
+The following four strategies are commonly used in cyber risk management.
+
+- _Acceptance_: acceptance occurs when the result after a cost/benefits analysis shows that the costs of prevention and protection would outweigh the possible cost of loss due to the risk.
+- _Mitigation_: mitigation is when controls and other preventive measures are used to reduce the level of risk facing an organization.
+- _Risk Transfer_: risk transfer occurs when one party shifts the pure risk from one party to another.
+- _Avoidance_: avoidance occurs when a party eliminates risks and hazards that can negatively affect an organization and its assets.
+
+## Relevant Courses:
```
Comment (SSwiniarski) [inline content/cybersecurity/concepts/risk-management/risk-management.md:24]: Course links will be provided with the Metadata.

File: content/cybersecurity/concepts/spear-phishing/spear-phishing.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Spear phishing
```
Comment (SSwiniarski) [inline content/cybersecurity/concepts/spear-phishing/spear-phishing.md:1]: We're missing the metadata for this entry. Look at the [source here](https://github.com/Codecademy/docs/blob/main/content/cybersecurity/concepts/cyber-attack/cyber-attack.md) to see an example of what we need.

Also, remove the h1 header, the metadata title will provide that.

File: content/cybersecurity/concepts/spear-phishing/spear-phishing.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+Spear phishing is a type of social engineering scam that attempts to trick the recipient into providing confidential information to the attacker. Spear phishing is a more targeted cyber attack than phishing.
```
Comment (SSwiniarski) [inline content/cybersecurity/concepts/spear-phishing/spear-phishing.md:3]:
```suggestion
**Spear phishing** is a type of social engineering scam that attempts to trick the recipient into providing confidential information to the attacker. Spear phishing is a more targeted cyber attack than phishing.
```

File: content/cybersecurity/concepts/spear-phishing/spear-phishing.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+- _Links, addresses and domains_: look for discrepancies in email addresses, links and domain names to identify a potential spear phishing attempt. If there is a link, hover over it to reveal the URL. 
+
+## How to prevent spear phishing?
+
+A multi-faceted approach is the best way to prevent spear phishing. Email protection solutions can detect suspicious emails. Security awareness training is equally as important. Employees are often the first line of defense so it is very important that they are trained on how to detect suspicious cyber activity. Employees need to know that they should not open spam or unsolicited email from unknown parties and that they should never click on links in the email. 
+
+## Spear phishing vs. Phishing vs. Whaling:
+
+Spear phishing often is used interchangeably with phishing and whaling, but it is important to note the differences. Phishing is a more generic attempt to dupe a victim. It emphasizes quantity. Phishing emails or texts are usually sent to large groups of individuals as opposed to a targeted individual in a spear phishing campaign. Whaling prioritizes high-level targets, specifically the various chief officers of an organization.
```
Comment (SSwiniarski) [inline content/cybersecurity/concepts/spear-phishing/spear-phishing.md:23]: You mention the comparison to whaling, but never define the term in opposition to spear phishing. There should be a sentence covering it.

File: content/cybersecurity/concepts/spear-phishing/spear-phishing.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+## How to prevent spear phishing?
+
+A multi-faceted approach is the best way to prevent spear phishing. Email protection solutions can detect suspicious emails. Security awareness training is equally as important. Employees are often the first line of defense so it is very important that they are trained on how to detect suspicious cyber activity. Employees need to know that they should not open spam or unsolicited email from unknown parties and that they should never click on links in the email. 
+
+## Spear phishing vs. Phishing vs. Whaling:
+
+Spear phishing often is used interchangeably with phishing and whaling, but it is important to note the differences. Phishing is a more generic attempt to dupe a victim. It emphasizes quantity. Phishing emails or texts are usually sent to large groups of individuals as opposed to a targeted individual in a spear phishing campaign. Whaling prioritizes high-level targets, specifically the various chief officers of an organization. 
+
+## Relevant Courses:
```
Comment (SSwiniarski) [inline content/cybersecurity/concepts/spear-phishing/spear-phishing.md:25]: Course links will be provided by the metadata.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@realber I did a first review, and left a couple of comments. The most important is we need to resolve the lack of metadata in these entries.

Also, because this was pushed from your main branch, you may want to wait until this is merged before submitting a new PR.

**Comment by SSwiniarski** [comment]
Closed per @jonchery

File: content/cybersecurity/concepts/spear-phishing/spear-phishing.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+  - 'Cybersecurity'
+Tags:
+  - 'Cybersecurity'
+  - 'Validation'
+CatalogContent:
+  - 'introduction-to-cybersecurity'
+  - 'paths/fundamentals-of-cybersecurity'
+---
+
+**Spear phishing** is a type of social engineering scam that attempts to trick the recipient into providing confidential information to the attacker. Spear phishing is a more targeted cyber attack than phishing.
```
Comment (yangc95) [inline content/cybersecurity/concepts/spear-phishing/spear-phishing.md:15]: Good op to link out to cyber attack entry: https://www.codecademy.com/resources/docs/cybersecurity/cyber-attack

File: content/cybersecurity/concepts/spear-phishing/spear-phishing.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+
+## Spear phishing In-Use:
+
+The targeted nature of spear phishing makes it difficult to detect. For example, attackers who claim to be the CEO could trick finance executives into sending money to their bank account. An attacker might create messages that seem like IT is asking for more information. Deploying multi-factor authentication, strong password management policies, and training and awareness campaigns are good measures to help mitigate the risk.
```
Comment (yangc95) [inline content/cybersecurity/concepts/spear-phishing/spear-phishing.md:19]:
```suggestion
The targeted nature of spear phishing makes it difficult to detect. For example, attackers who claim to be the CEO could trick finance executives into sending money to their bank account. An attacker might create messages that seem like the IT (Information Technology) Department is asking for more information. Deploying [multi-factor authentication](https://www.codecademy.com/resources/docs/cybersecurity/multi-factor-authentication), strong password management policies, and participating in training and awareness campaigns are good measures to help mitigate risks.
```

File: content/cybersecurity/concepts/spear-phishing/spear-phishing.md [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+- _Links, addresses and domains_: look for discrepancies in email addresses, links and domain names to identify a potential spear phishing attempt. If there is a link, hover over it to reveal the URL. 
+
+## How to prevent spear phishing?
+
+A multi-faceted approach is the best way to prevent spear phishing. Email protection solutions can detect suspicious emails. Security awareness training is equally as important. Employees are often the first line of defense so it is very important that they are trained on how to detect suspicious cyber activity. Employees need to know that they should not open spam or unsolicited email from unknown parties and that they should never click on links in the email. 
+
+## Spear phishing vs. Phishing vs. Whaling:
+
+Spear phishing often is used interchangeably with phishing and whaling, but it is important to note the differences. Phishing is a more generic attempt to dupe a victim. It emphasizes quantity. Phishing emails or texts are usually sent to large groups of individuals as opposed to a targeted individual in a spear phishing campaign. Whaling prioritizes high-level targets, specifically the various chief officers of an organization.
```
Comment (yangc95) [inline content/cybersecurity/concepts/spear-phishing/spear-phishing.md:35]:
```suggestion
Spear phishing is often used interchangeably with phishing and whaling, but it is important to note the differences. Phishing is a more generic attempt to dupe a victim. It emphasizes quantity. Phishing emails or texts are usually sent to large groups of individuals as opposed to a targeted individual in a spear phishing campaign. Whaling prioritizes high-level targets, specifically the various chief officers of an organization.
```

**Review by yangc95 — COMMENTED** [review]
@realber thanks for these entries! I left a few comments 🙂 when you're finished please let me know by tagging me and we can see if it's ready to merge

**Comment by jonchery** [comment]
@yangc95 

@realber is no longer working on this project. That said, he won't be able to make any of the requested changes. Since I'm not the author of this PR, I won't be able to submit or make those changes too.

**Comment by SSwiniarski** [comment]
@yangc95, @jonchery I committed all the suggested changes. If @yangc95 approves them, I can go ahead and merge this.

**Review by yangc95 — APPROVED** [review]
@SSwiniarski LGTM

**Review by SSwiniarski — APPROVED** [review]

**Comment by yangc95** [comment]
@SSwiniarski Thanks for merging!

@realber here's the live links: 
- https://www.codecademy.com/resources/docs/cybersecurity/risk-management
- https://www.codecademy.com/resources/docs/cybersecurity/spear-phishing

Screen Shot 2023-03-08 at 3 29 12 PM
Screen Shot 2023-03-08 at 3 30 05 PM

> It looks as if you're anonymous. If you want to be credited follow the instructions below to add your GitHub username to your Codecademy profile. If not, there is nothing more you need to do.
> 
> To add your GitHub username to your Codecademy profile:
> 
> 1.) Add your GitHub Username to your Codecademy Profile Details, you can get there on your desktop by pressing on your profile picture in the upper right-hand corner and selecting "Profile" in the drop-down box:
> 
> image
> 
> 2.) Then log out and log back into Codecademy using your GitHub login:
> 
> githublogin
>
> If the steps above do not work, it may be due to an email or local configuration issue.

**Comment by jonchery** [comment]
Hi @yangc95 , unfortunately, @realber is not longer with us and won't be able to make those changes.

**Comment by yangc95** [comment]
@jonchery would you mind me asking how you became aware of @realber's absence? Thanks for any info that can be disclosed!

**Comment by jonchery** [comment]
He was my content contributor (CC) working on the doc (as well as several others). Once the doc project was done, the contract was complete.

**Comment by yangc95** [comment]
🙂 @jonchery thank you for this transparency. In which case, I will go ahead and make these changes unless told otherwise

**Pull request merged**